medmodels/medrecord/medrecord.py

```diff
@@ -1381,10 +1371,11 @@ def _describe_group_nodes(
                 ),
             }
 
-        if not add_ungrouped:
+        if groups:
             return nodes_info
 
-        ungrouped_count = len(nodes) - len(set(grouped_nodes))
+        # If no groups were specified, add ungrouped nodes
```

Comment:
Is this comment supposed to be here? If yes, I'd remove it. It makes it more confusing than without